sonar-project.properties

```diff
@@ -1,7 +1,7 @@
 sonar.projectKey=NHSDigital_NRLF
 sonar.organization=nhsdigital
+sonar.projectName=NRLF
 sonar.python.version=3.9.5
 sonar.terraform.provider.aws.version=4.63.0
-sonar.coverage.exclusions=api/tests/**, tests/**, api/**/tests/**, feature_tests/**, cron/seed_sandbox/tests/**, data_contracts/**/tests/**, firehose/tests/**, firehose/processor/tests/**, helpers/tests/**, mi/**/tests/**
```

Comment:
Should we keep the `sonar.coverage.exclusions` as well?

Reply:
we dont do a coverage analysis in sonarcloud